docs/specs/ops.yml

```diff
@@ -282,3 +282,82 @@ outputs:
   .errors.log: |
     {"message_severity":"error","code":"violate-schema","message":"Error converting value \"a/b/.openpublishing.redirection.test.json\" to type 'System.Collections.Generic.HashSet`1[Microsoft.Docs.Build.PathString]'.","file":".openpublishing.publish.config.json","line":3,"column":66}
     {"message_severity":"error","code":"violate-schema","message":"Error converting value \"a/b/.openpublishing.redirection.test.json\" to type 'System.Collections.Generic.HashSet`1[Microsoft.Docs.Build.PathString]'.","file":".openpublishing.publish.config.json","line":3,"column":66}
+
+---
+# provide source url in .publish.json for content, toc and resource pages but not redirection pages.
```

Comment:
Since these tests are about `.publish.json`, move them to `output.yml` which tests all other aspects of `.publish.json` file.